FordLabs/retroquest
# Add header to ChangeTeamDetailsPage and ResetPasswordPage

## Overview
This is continued work towards the endeavour of adding email and password reset functionality

## What was done
- [x] Create a header component with just the retroquest logo, and the ability to pass in a name and children
- [x] Use that compnent within the team header, and within the ChangeTeamDetailsPage, and ResetPasswordPage pages

### Demo
<img width="1588" alt="Screen Shot 2022-09-14 at 10 21 50 AM" src="https://user-images.githubusercontent.com/17144844/190180926-96da9f22-ccce-49a4-8617-d43d0ddd8de9.png">
<img width="1396" alt="Screen Shot 2022-09-14 at 10 21 59 AM" src="https://user-images.githubusercontent.com/17144844/190180954-cdf8e38a-4e54-4ad3-906b-390ea43057a5.png">

## Testing Instructions
1) Ensure ChangeTeamDetailsPage and ResetPasswordPage both have a header now with the text RetroQuest, and the retroquest logo that links back to the login page
2) Ensure that once you log in, you see the same header as you did before, with the team name, logo, and all the nav options
